### PR TITLE
[FSDE-836] Added support of Skeleton component

### PR DESCRIPTION
**Pull request by janibolkvadze**

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-68/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://5b047daac3a2ed8b785803ca349d8f23b5f99ce3-chi-pr.enterprise.ctl.io

File: src/chi/components/skeleton/skeleton.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+);
+
+.chi-skeleton {
+  animation: shine $skeleton-animation-duration infinite linear;
+  height: 2rem;
+  @include background-gradient;
+
+  &.-circle,
+  &.-square {
+    @each $type in map-keys($skeleton-sizes) {
```
Comment (jllr) [inline src/chi/components/skeleton/skeleton.scss:42]: @janibolkvadze this line should go before line 46. Please change

File: src/website/views/components/skeleton/index.pug [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Skeleton
+description: Use skeleton to achieve loading state visual for asynchronously rendered UI elements
```
Comment (jllr) [inline src/website/views/components/skeleton/index.pug:3]:
```suggestion
description: Use skeleton to achieve loading state visual for asynchronously rendered UI elements.
```

File: src/website/views/components/skeleton/_examples.pug [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      li.-disabled
+        a(href=`#webcomponent-square`) Web Component
+      li.-active
+        a(href=`#html-square`) HTML Blueprint
+  .chi-tabs-panel#webcomponent-square
+  .chi-tabs-panel.-active#html-square
+    :code(lang="html")
+      <div class="chi-skeleton -square"></div>
+
+h3 Sizes
```
Comment (jllr) [inline src/website/views/components/skeleton/_examples.pug:63]: @janibolkvadze please change example sizes to be consisten with documentation order (i.e: base, rounded, circle and square)

**Review by jllr — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-68/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://534187acbc7cdbe68571ad5c1efca5342c3b0fcf-chi-pr.enterprise.ctl.io

**Review by jllr — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-68/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://abe741dd9b0926adbd54a80a19560f3d3aec3fd0-chi-pr.enterprise.ctl.io

**Pull request merged**